pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -621,6 +631,10 @@ private void addNewRow(int docId, GenericRow row) {
           case DOUBLE:
             forwardIndex.add(((Number) value).doubleValue(), -1, docId);
             break;
+          case BIG_DECIMAL:
```

Comment:
The above comment no longer apply.
We should probably add some comment about using byte[] to support `BIG_DECIMAL`. It works because `BIG_DECIMAL` is actually stored as byte[] underlying